docs/setup/Panacea-ModLoader/index.md

```diff
@@ -9,124 +9,125 @@
-* **[KH2 Rando Tracker](https://github.com/Dee-Ayy/KH2Tracker/releases/latest)** - Not an OpenKH mod but instead a full fledged automated tracker program for the Important Checks in game. Checkout [Hint Systems](https://kh2rando.com/hints) for info about different ways to play Rando!
+> [!TIP]
+> I heavily recommend you install the below two mods to help provide a better experience when playing rando:
+> * `KH2FM-Mods-equations19/auto-save` - This mod auto saves the game for you as you enter rooms. Be sure to make at least 1 regular save in game, then if you ever crash or your game closes unexpectedly, just hold the `SELECT` button while loading a save, and the auto-save will be loaded instead.
+> * `KH2FM-Mods-equations19/soft-reset` - Hold `L1+L2+R1+R2+Start` at the same time to immediately reset the game to the start screen. Very useful if you accidentally softlock in boss/enemy rando, or just to restart the game faster!
+> * [KH2 Rando Tracker](https://github.com/Dee-Ayy/KH2Tracker/releases/latest) - Not an OpenKH mod but instead a full fledged automated tracker program for the Important Checks in game. Checkout [Hint Systems](https://kh2rando.com/hints) for info about different ways to play Rando!
 
 You Mods Manager should look like this when you are done:
 
 ![Final View](../images/Panacea-ModLoader/Final%20View.png)
 
 # *You are now ready to play the KH2 Randomizer!*
+Please use the #help channel in our [Community Discord](https://discord.gg/vKhdwNAmzE) if you have any errors or questions.
 
 __Technical And Stability Notes__
-1. As mentioned earlier in the guide, the Mods Manager/Lua doesn't seem to work if the folder is under a "OneDrive" directory. We don't know exactly why but it's a consistent thing between users. Be sure to move your folders and re-run the setup wizards if you installed in a "OneDrive" directory.
-2. Recommended to set FPS limit to **60**.
-3. Recommended to run game in Windowed/Borderless Fullscreen mode. Fullscreen is stable but the game can crash if you alt-tab out.
-4. Overlays that hook onto the game process (Steam, RivaTuner, NVIDIA Overlay, etc.) and recording programs (OBS, Streamlabs, Xsplit) seem to decrease stability when playing rando. It's recommended to disable them, or in the case of recording programs use window/desktop capture, in order to minimize your chances of a crash.
+1. As mentioned earlier in the guide, the Mods Manager/Lua doesn't seem to work if the folder is under a "OneDrive" directory. We don't know exactly why but it's a conistent thing between users. Be sure to move your folders and re-run the setup wizards if you installed in a "OneDrive" directory.
+2. Recommended to set fps limit to `60fps`.
+3. Recommended to run game in windows/borderless windowed mode. Fullscreen is stable but the game can crash if you alt-tab out.
```

Comment:
The options are "Windowed" and "Borderless Fullscreen".

![image](https://github.com/tommadness/KH2Randomizer/assets/2530991/eb615ccc-4a51-4ea9-8e43-3fde6a8e4a43)